new-lamassu-admin/src/components/inputs/base/Autocomplete.js

```diff
@@ -14,6 +14,7 @@ const Autocomplete = ({
   multiple,
   onChange,
   labelProp,
+  keepOpenProp,
```

Comment:
Shouldn't this `keepOpenProp` be also on the `<MAutoComplete>`  node? It works as it currently is, but other props are being explicitly passed on the node. It's more for consistency sake